web/webkit/src/main/resources/toserve/lift.js

```diff
@@ -638,6 +642,12 @@
   })();
 
   window.liftJQuery = {
+    onEvent: function(elementOrId, eventName, fn) {
+      if (typeof elementOrId == 'string')
+        elementOrId = '#' + elementOrId;
+
```

Comment:
I don't know about you but not using braces in JavaScript makes me nervous.

I also know it's easy to overlook when switching from Scala.